chapter02/dtj/ex3

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/perl
+
+use warnings;
+
+print "Enter the radius: ";
+chomp($radius = <STDIN>);
+
```

Comment:
- Looks good!  Probably worth adding some validation checks for non-numeric data.